docs/examples/101/redis-cache/main.bicep

```diff
@@ -0,0 +1,21 @@
+param environmentName string
+param redisCacheSku string
+param redisCacheFamily string
+param redisCacheCapacity int
+param location string
+
+var redisCacheName = concat(environmentName, uniqueString(resourceGroup().id))

```

Comment:
Going forward pleas try to use string interpolation syntax instead of `concat()`, so this line would change to

```
var redisCacheName = '${environmentName}${uniqueString(resourceGroup().id)}'
```